soc/arm/st_stm32/stm32f7/Kconfig.soc

```diff
@@ -15,6 +15,9 @@ config SOC_STM32F723XE
 config SOC_STM32F746XG
 	bool "STM32F746XG"
 
+config SOC_STM32F756XG
```

Comment:
Rename to STM32F756XX, since recent addition of flash an dram size in device tree files, we don't need last digit information in Kconfig.

Reply:
Changed as requested, also renamed the string on the line below and and renamed soc/arm/st_stm32/stm32f7/Kconfig.defconfig.stm32f756xg to soc/arm/st_stm32/stm32f7/Kconfig.defconfig.stm32f756xx.

Reply:
Almost forgot - also changed the Kconfig.board dependency to: "depends on SOC_STM32F756XX"